iOS-Weather-App/Controllers/CurrentDetailedWeatherViewController/CurrentDetailedWeatherViewController.swift

```diff
@@ -0,0 +1,40 @@
+//
+// Created by pavels.garklavs on 23/02/2022.
+//
+
+import UIKit
+
+class CurrentDetailedWeatherViewController: UIViewController {
+
+    private let detailsTableView = UITableView()
+
+    override func viewDidLoad() {
+        setupTableView()
+    }
+}
+
+private extension CurrentDetailedWeatherViewController {
+    func setupTableView() {
+        view.addSubview(detailsTableView)
+        detailsTableView.delegate = self
+
+        detailsTableView.snp.makeConstraints { make in
+            make.left.right.equalToSuperview()
```

Comment:
don't use left, right pls, remove it

Reply:
you can set all properties at 1 line, like make.top.bottom.leading.bottom.equal....